test/arm/neon/rndn.c

```diff
@@ -5,14 +5,11 @@
 
 static int
 test_simde_vrndnh_f16 (SIMDE_MUNIT_TEST_ARGS) {
+#if 1
   struct {
     simde_float16_t a;
     simde_float16_t r;
   } test_vec[] = {
-    #if !defined(SIMDE_FAST_NANS)
-        {            SIMDE_NANHF,
-                     SIMDE_NANHF, },
-    #endif
     { SIMDE_FLOAT16_VALUE(    -1.50),
```

Comment:
Why was this removed?

Reply:
Because the `test_simde_vrndn_f16` function has already been tested with the input `SIMDE_NANHF`.